gooey-gui/app/styles/custom.css

```
@@ -177,8 +177,18 @@ a:hover {
-    font-family: avenir-lt-w01_85-heavy1475544,avenir-lt-w05_85-heavy,"avenir-lt-w05_85-heavy","Space Grotesk", sans-serif;
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.table > thead > tr > th,
+strong,
+.bold,
+.semibold {
+  font-family: avenir-lt-w01_85-heavy1475544, avenir-lt-w05_85-heavy,
```

Stylelint (17.14.0)

[error] Expected quotes around "avenir-lt-w01_85-heavy1475544" (font-family-name-quotes)

(font-family-name-quotes)